ONSdigital/ras-maintenance-page
# Adding unplanned maintenance page

# Motivation and Context
Currently we only have one message when we need to put up the maintenance page. For normal deployments, this message is fine but in the case of unplanned maintenance, the message needs to be updated. This PR changes the ras-maintenance-page repo from just staticfile to a small python app that chooses which maintenance file to display depending on the environment variable. 

# What has changed
- RAS-maintenance-page is now a small python/flask app
- Added unplanned maintenance page
- Added tests for the code
- Reorganised the files and folders
- Added travis-ci
- Updated the readme

# How to test?
- Run the app, currently it's set to use `localhost:8077`. When running without the environment variable `MAINTENANCE_TEMPLATE`, it should display the `Service temporarily unavailable` page. When the variable is set with `UNPLANNED_MAINTENANCE`, it'll display the `Unplanned maintenance` page.
- You can try and use it on devtest environment. If you click on the link, it'll take you to the `Service temporarily unavailable`.  When you set the environment variable and restart the app, it should take you to the `Unplanned maintenance` page.
- Run the tests

# Links
[Trello](https://trello.com/c/TW0oaRyT)
